client/lib/wpcom-undocumented/lib/site.js

```diff
@@ -217,6 +217,17 @@ UndocumentedSite.prototype.newExport = function( fn ) {
 };
 
 /**
+ * Returns media storage limits and space used for a given site. If site has
+ * unlimited storage or is a jetpack site, values returned will be -1.
+ *
+ * @param {Function} callback - called on completion of the GET request
+ * @return {Object} promise - resolves on completion of the GET request
+ */
+UndocumentedSite.prototype.getMediaStorage = function( callback ) {
```

Comment:
Renamed in 7d572c2. As a general observation I find it unclear to have a chain like that as I wouldn't know what it returns at any point of the chain. In terms of rest clients I would expect it as a property:

eg:
```javascript
wpcom.undocumented.site.mediaStorage.get( { siteId: siteId } ); 
```
versus:
```javascript
wpcom.undocumented().site( siteId ).mediaStorage().get();
```